plugins/playlist/src/components/PlaylistSortPicker/PlaylistSortPicker.test.tsx

```diff
@@ -40,13 +40,24 @@ describe('PlaylistSortPicker', () => {
         getByRole(getByTestId('sort-picker-select'), 'button'),
       );
     });
-    const abcSort = getByText('Alphabetical');
+
+    const abcSort = getByText('A-Z (ascending)');
     expect(abcSort).toBeInTheDocument();
 
     fireEvent.click(abcSort);
     await waitFor(() => {
       expect(updateSort).toHaveBeenLastCalledWith(
-        DefaultSortCompareFunctions[DefaultPlaylistSortTypes.alphabetical],
+        DefaultSortCompareFunctions[DefaultPlaylistSortTypes.ascending],
```

Comment:
I would recommend checking that the items are rendered in the correct order: i.e. that Ambrish is the first item and Ramachandiram the second for ascending order and the opposite for descending order. Just checking that the function was called does not guarantee that its implementation is correct.

Reply:
@camilaibs modified test case please check

Reply:
I was actually thinking of something like this:
```tsx
const ascendingItems = getAllByText(/playlist-/);
// expect 1st item is playlist-1
expect(ascendingItems[0]).toHaveTextContent('playlist-1');
// expect 2nd item is playlist-2
expect(ascendingItems[1]).toHaveTextContent('playlist-2');
```